sample/src/main/java/com/prolificinteractive/materialcalendarview/sample/DynamicSettersActivity.java

```diff
@@ -24,6 +27,7 @@
 
     @InjectView(R.id.textView) TextView textView;
     @InjectView(R.id.calendarView) MaterialCalendarView widget;
+    private int currentTileSize = 44;
```

Comment:
Instead of a Java constant, I modified the way the library defines the default value in XML. This way we can later define other values for tablets or something.